docs/barbican-kms-plugin/using-barbican-kms-plugin.md

```diff
@@ -23,7 +23,7 @@ It is recommended to read following kubernetes documents
 The following installation steps assumes that you have a Kubernetes cluster(v1.10+) running on OpenStack Cloud.
 
 
-### Create 256bit(32 byte) cbc key and store in barbican
+### Create 256-bit(32 byte) cbc key and store in barbican
```

Comment:
```suggestion
### Create 256-bit (32 byte) CBC key and store in barbican
```

maybe it makes sense to fix CBC in other places, and other modes of operation.

Reply:
Yeah, good observation, I will update soon.